main.py

```diff
@@ -0,0 +1,90 @@
+class Node:
+    # This class provides a generic node for a linked list. 
+    def __init__(self, number, name):
+        self.value = int(number) #number element, in this case ssn
+        self.name = str(name) #name element attached to ssn
+        self.next = None #pointer
+
+class IRS_Single_Linked_List:
+    def __init__(self):
+        self.headPtr = None
+
+    def insert_at_end(self, number, name):
+        new_node = Node(number, name)
+        if self.headPtr is None:
+            self.headPtr = new_node
+            return
+        n = self.headPtr
```

Comment:
Avoiding using variable names like `n`, `temp`. While it is okay to use it when teaching or drawing out a plan of work, it isn't a good practice in the real world. Make sure your variables convey meaning as to what it stores. For instance, `n` here would be your `iterative_node`.